packages/coinstac-ui/app/render/components/computation-field-basic.js

```diff
@@ -7,54 +7,52 @@ import {
   HelpBlock,
 } from 'react-bootstrap';
 
-const INPUT_REF = 'computation-field-input';
-
 export default class ComputationFieldBasic extends Component {
-  constructor(props) {
-    super(props);
-    this.handleButtonClick = this.handleButtonClick.bind(this);
-  }
-
   handleButtonClick(increment) {
-    const { max, min, onChange } = this.props;
-    const value = this.refs[INPUT_REF].props.value;
+    const {
```

Comment:
Method cleaned up to rely on Redux Form’s passed down properties instead of the homemade _refs_ approach.